fnf_converter.py

```diff
@@ -340,17 +341,19 @@ def getNotesFromPlayer(self, player_id: int, keys_count: int):
 				
 				# Add the note on the list if it's for the player
 				if is_player_section and note[1] < keys_count:  # the section is for the player, then get only columns 0 to keys_count-1
-					note[0] = int(note[0])  # round and apply offset
+					note[0] = int(note[0])
 					note[2] = int(note[2]) if (int(note[2]) >= 0) else 0  # round and set 0 if the length is negative
 					notes_list.append(note)
 				elif not is_player_section and ((player_id == 0) or (note[1] >= keys_count)):  # the section isn't for the player, exclude columns 0 to keys_count-1 OR we want all notes regardless of the player
-					note[0] = int(note[0])  # round and apply offset
+					note[0] = int(note[0])
 					note[1] %= keys_count  # apply modulo keys_count
 					note[2] = int(note[2]) if (int(note[2]) >= 0) else 0  # round and set 0 if the length is negative
 					notes_list.append(note)
-			   
-		notes_list.sort(key=lambda note: ' '.join(map(str, note[:3]))) # sort the notes by offset order (using the index 0) and ignore any argument, if argument count exceeds 3
 
+		# notes_list.sort(key=lambda note: ' '.join(map(str, note[:3]))) # sort the notes by offset order (using the index 0) and ignore any argument, if argument count exceeds 3
+		# ^removed as it may geenrate memory leaks and fill your RAM wtf (on Linux with kubuntu 23.10)

```

Comment:
Ah, thanks for letting me know. I havent tested it on Linux, but good to know that you catched that